README.md

```diff
@@ -21,11 +21,23 @@
   </p>
 </p>
 
-# Prerequisites
+# 👉  Prerequisites
 
-- macOS with Sketch 41+
+- macOS with Sketch 42+
```

Comment:
Bumped because I saw this in a thread on Sketch Dev forum that we need Sketch 42